yoti_python_sdk/__init__.py

```diff
@@ -9,11 +10,18 @@
     'YOTI_API_VERSION': 'v1',
 }
 
+main_ns = {}
+ver_path = convert_path('yoti_python_sdk/version.py')
```

Comment:
So, do we read this file only once, at startup?

Reply:
We use it again when sending the headers: `sdk_version = yoti_python_sdk.__version__`, but it's not possible to access the version that way in this class, as it hasn't been initalised yet

Reply:
Can this be refactored so that we only read from disk once, when the client is initialised?  
More File IO each time we request a profile really isn't a good thing.

Reply:
`__init.py__` is only run once, to initialize the package ([link to docs](https://docs.python.org/3/tutorial/modules.html#packages)). From then on, it's part of a variable which is available package-wide, but we only read from disk once